hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/AbstractFSWAL.java

```diff
@@ -414,6 +485,23 @@ protected final int getPreallocatedEventCount() {
     return floor << 1;
   }
 
+  protected void setWaitOnShutdownInSeconds(int waitOnShutdownInSeconds,
```

Comment:
@Apache9  ,  thank you very much for rebase,  LGTM,   just a little problem, may be we could declare this method as final  because it is called by ctor ?

Reply:
Let me check.